index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+    <title>Project 1: Me Button</title>
+
+    <canvas id="canvas" width="800" height="200"></canvas>
+
+    <script src="js/script.js" type='text/javascript'></script>
```

Comment:
If you are maintaining a line between body tags like you do at the top please add a line here.
```suggestion
    <script src="js/script.js" type='text/javascript'></script>
    
```